<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Email</class>
 <widget class="QWidget" name="Email">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收件箱</string>
  </property>
  <widget class="QPushButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>591</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="DelEmailBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除所选邮件</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
